src/pages/PatientPersonalInfo.js

```diff
@@ -153,24 +175,23 @@ const PatientPersonalInfo = ({
                   value={intakeState.emailId}
                 />
               ) : info.field === "height" ? (
-                <>
-                  <div className="height-wrapper">
-                    {HEIGHT.map((height) => {
-                      <label>{info.title}</label>;
-                      return (
-                        <>
-                          <input
-                            className="bp"
-                            type="text"
-                            name={info.field}
-                            placeholder={height === "feet" ? "Ft." : "In"}
-                            onChange={handleValidateNumbers}
-                          />
-                        </>
-                      );
-                    })}
-                  </div>
-                </>
+                <div className="height-wrapper">
+                  {HEIGHT.map((height) => {
```

Comment:
it's better to split "Ft." and "In" inputs, no need to loop 2 items and add multiple conditions to identify which input it is

Reply:
this change has been made